app/lib/katello/validators/alternate_content_source_products_validator.rb

```diff
@@ -7,7 +7,7 @@ def validate_each(record, attribute, value)
             product = ::Katello::Product.find(value)
             content_type = record.alternate_content_source.content_type
             if product.repositories.with_type(content_type).has_url.empty?
-              record.errors[attribute] << N_("The product %{name} has no %{type} repositories with upstream URLs to add to the alternate content source.") % { name: product.name, type: content_type }
+              record.errors.add(attribute, N_("%{name} has no %{type} repositories with upstream URLs to add to the alternate content source.") % { name: product.name, type: content_type })
```

Comment:
Do you think this should be `N_` or `_`?

Reply:
Not entirely sure, but documentation suggests N_ for dynamic translations..
https://github.com/grosser/gettext_i18n_rails#unfound-translations-with-rake-gettextfind

Reply:
Just tested it out. 

With `N_`
![Screenshot at 2023-07-13 17-11-28](https://github.com/Katello/katello/assets/2593270/de038a83-9204-49ba-9429-0fe7b991e90f)

With `_`
![Screenshot at 2023-07-13 17-14-12](https://github.com/Katello/katello/assets/2593270/f9eb5e6e-a1b5-46ef-8ddc-fc05aa7db4d2)